packages/neptune-extension/src/common/components/checkout-modal/CheckoutModal.tsx

```diff
@@ -26,7 +26,7 @@ const CheckoutModal: React.FC<CheckoutModalProps> = ({
    */
 
   async function handleSubmit() {
-    const content = await platformNotebook.getContent();
+    const content = await platformNotebook.saveWorkingCopyAndGetContent(); // FIXME: This will be removed anyway.
```

Comment:
Why this will be removed anyway?

Reply:
As I understand checkout modal is already merged.
Could you please rebase this branch and fix the code so we can see final result?

Reply:
Sure here you have.